app/frontend/src/pages/Projects/Projects.tsx

```diff
@@ -1,6 +1,32 @@
+import { useEffect, useState } from "react";
+import { getAllGithubRepos } from "../../api/api";
+
 const Projects = () => {
+    const [repos, setRepos] = useState([]);
+
+    const fetchAllGithubRepos = async () => {
```

Comment:
Since I am in Projects folder
this functions will get all the repos, inside each repo is a project or is each repo is a project?
Sorry it is a little confusing for me

Reply:
each repo is each project inside of Projects page where I have a list of projects/repos fetched from Github